### PR TITLE
Add doc how to build a registry image with VM disk

### PR DESCRIPTION
**Pull request by rollandf**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: doc/image-from-registry.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Creating a registry image with a VM disk
+The purpose of this document is to show how to create registry image containing a Virtual Machine image that can be import into a PV.
+
+## Prerequisites
+The VM disk image should be called 'disk.img' and placed in the root folder.
+
+## Create an image with Buildah
```
Comment (igoihman) [inline doc/image-from-registry.md]: could you add a few words about Buildah and refer to its documentation?

File: doc/image-from-registry.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Add the VM disk to the image and commit:
+
+```bash
+$ buildah copy $newcontainer ./disk.img /
+buildah commit $newcontainer my-vm-img
+```
+
+To push an image from your local Buildah container storage, check the image name, then push it it using the buildah push command. Remember to identify both the local image name and a new name that includes the location (<registry>:5000, in this case):
+```
+buildah push --tls-verify=false my-vm-img:latest <registry>:5000/my-vm-img:latest
```
Comment (zvikorn) [inline doc/image-from-registry.md]: At what point did you tag the image with 'latest'? during the commit? this part is missing

File: doc/image-from-registry.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+```
+FROM scratch
+ADD disk.img /
+```
+
+Build, tag and push the image:
+
+```bash
+$ docker build . -t my-vm-img
+$ docker tag my-vm-img <registry>:5000/my-vm-img
```
Comment (zvikorn) [inline doc/image-from-registry.md]: Don't you want to tag it with 'latest' as well?

File: doc/image-from-registry.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Creating a registry image with a VM disk
+The purpose of this document is to show how to create registry image containing a Virtual Machine image that can be import into a PV.
+
```
[the remark this answers is not in the document]
Comment (rollandf) [reply doc/image-from-registry.md]: Done

File: doc/image-from-registry.md (hunk 1 above)
Comment (rollandf) [reply doc/image-from-registry.md]: Done

File: doc/image-from-registry.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+## Prerequisites
+The VM disk image should be called 'disk.img' and placed in the root folder.
+
+## Create an image with Buildah
+
+Create a new container from scratch and keep its name in a variable:
```
[the remark this answers is not in the document]
Comment (rollandf) [reply doc/image-from-registry.md]: Done

File: doc/image-from-registry.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+```
+
+Add the VM disk to the image and commit:
+
+```bash
+$ buildah copy $newcontainer ./disk.img /
+buildah commit $newcontainer my-vm-img
+```
+
+To push an image from your local Buildah container storage, check the image name, then push it it using the buildah push command. Remember to identify both the local image name and a new name that includes the location (<registry>:5000, in this case):
```
[the remark this answers is not in the document]
Comment (rollandf) [reply doc/image-from-registry.md]: Done

File: doc/image-from-registry.md (hunk 2 above)
Comment (rollandf) [reply doc/image-from-registry.md]: latest tag is applied automatically

File: doc/image-from-registry.md (hunk 3 above)
Comment (rollandf) [reply doc/image-from-registry.md]: It will get it automatically

**Comment by rollandf** [comment]
ci test please

**Comment by rollandf** [comment]
ci test please

**Comment by rollandf** [comment]
ci test please

**Pull request merged**